sopel/plugin.py

```diff
@@ -129,7 +131,7 @@
 """
 
 
-def unblockable(function):
+def unblockable(function: typing.Any) -> typing.Any:
```

Comment:
This seems like it should be as below, but I must be missing something.

```suggestion
def unblockable(function: typing.Callable) -> typing.Callable:
```

Reply:
Oh, I know why! The reason is that you are not supposed to add an attribute to an object that doesn't define said attribute. It means that saying "I want a callable" here means mypy will complain that we set an undefined `unblockable` attribute.

That's why. Truly not the best we can do. You can put that in the "TODO: fix later" box.